specs/blocks.spec.ts

```diff
@@ -335,3 +335,27 @@ test('Desmos expressions panel is not visible.', async ({ page }) => {
   await page.waitForSelector('.dcg-grapher')
   await expect(page.locator('text=x=5')).not.toBeVisible()
 })
+
+test('Indentation class indents content correctly', async ({ page }) => {
```

Comment:
Good call on including tests for these style classes. Could we add them to a new `styles.spec.ts`, though?